server/src/game/server_actor.rs

```diff
@@ -0,0 +1,81 @@
+use std::collections::{HashMap, HashSet};
+
+use actix::prelude::*;
+
+use super::messages::*;
+
+// TODO: replace SessionInfo with a custom struct later
+pub type SessionInfo = HashMap<String, String>;
+pub type UserId = String; // TODO: Replace this with Uuid when custom struct is implemented
+
+// TODO: Move this to a dedicated file, probably
+struct Game {
+    players: HashSet<UserId>,
+    // TODO: Game state goes here
+}
+
+/// The game server which manages the active games.
+pub struct WsServer {
+    users: HashMap<UserId, Recipient<ServerMessage>>,
+    games: HashMap<String, Game>,
+}
+
+impl WsServer {}
+
+/// Makes WsServer an Actor, allowing it to send and receive messages from other actors
```

Comment:
I generally don't think rustdocs are necessary for `impl Trait for T` declarations, referring to the documentation for the trait and `T` is generally enough. I would only put one if something unintuitive is going on, but I imagine this is pretty standard for an `Actor` implementation.